src/server/index.js

```diff
@@ -106,6 +107,24 @@ const listen = (app) => {
     });
 };
 
+telemetry.initTelemetry({
+    enabled: true,
+    post: (url, data) => {
+        return new Promise((resolve, reject) => {
+            const request = https.request(url, { method: 'POST' }, (response) => {
+                resolve(response.statusCode);
+            });
+
+            request.on('error', (err) => {
+                reject(err);
+            });
+
+            request.write(data);
+            request.end();
```

Comment:
Hat-tip to @sarvaje for pointing out I used the old code from the VS Code extension that still had the "chunked" bug. Definitely need to refactor this so we can share the `post` implementation between `node` environments, but don't want to block this fix on publishing a new package to NPM.
```suggestion
            /*
             * Don't use request.write or request will be sent chunked.
             * Chunked requests break in node-based v2 Azure Functions,
             * which are used to run the webhint telemetry service.
             * https://github.com/Azure/azure-functions-host/issues/4926
             */
            request.end(data);
```